examples/gqlgen-service/server.go

```diff
@@ -32,6 +33,10 @@ func main() {
 		return next(ctx)
 	}
 	http.Handle("/query", handler.GraphQL(NewExecutableSchema(c)))
+	http.HandleFunc("/health", func(w http.ResponseWriter, r *http.Request) {
+		w.WriteHeader(http.StatusOK)
```

Comment:
`http.StatusOK` is sent by default if you start writing to `w`.

I also find it's nicer to use:
```
fmt.Fprintln(w, "OK")
```
so that if you `curl` it from the shell it writes a final newline

Reply:
Yeah I generally like to explicitly set the header anyway but swapped out for `fmt.Fprintln`